inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/ElasticsearchLoadNode.java

```diff
@@ -94,6 +97,7 @@ public ElasticsearchLoadNode(@JsonProperty("id") String id,
         this.index = Preconditions.checkNotNull(index, "index is null");
         this.documentType = documentType;
         this.primaryKey = primaryKey;
+        this.routingFieldName = primaryKey;
```

Comment:
`routingFieldName = primaryKey;`?

Reply:
the default value for inlong-es-connector

Reply:
@Oneal65 User can not set it. Maybe `tableOptions` method can set defaut value for `routing.field-name`.

Reply:
@gong OK